test/evergreen.yml

```diff
@@ -2490,6 +2496,20 @@ tasks:
             export "PATH=/opt/mongodbtoolchain/v3/bin:$PATH"
             ./workgen_perf_check.sh many-dhandle-stress.py output.log 50 500 500
 
+  - name: many-collection-test
+    commands:
+      - func: "fetch mongodb bin"
+      - func: "fetch mongo-tests repo"
+      - command: shell.exec
+        params:
+          working_dir: mongo-tests/largescale
+          script: |
+            set -o errexit
+            set -o verbose
+            export "PATH=/opt/mongodbtoolchain/v3/bin:$PATH"
+            pip3 install lorem pymongo
```

Comment:
I'd prefer using virtual environment instead of directly updating the system python module list. The mongodb toolchain contains the virtualenv binary according to [this](https://github.com/mongodb/mongo/blob/ffe55a9d08ecd09e70fb1298b3a37253a3be9cb4/etc/evergreen.yml#L7830).

Reply:
Sounds like a good approach, I will try that.

Reply:
f636523